src/convertToObject.js

```diff
@@ -3,16 +3,34 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ *
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleStorageObj = {};
+  const stylePropWithValues = sourceString.split('\n');
+  const stylePropWithValuesTrimmed = stylePropWithValues
+    .map(string => string
+      .slice(0, -1)
+      .trim()
+      .split(':')
+      .map(
+        entries => entries.trim()))
```

Comment:
If you're trimming each word in line 26, I guess you don't have to trim strings in line 24. But I don't know if it would work properly.

Reply:
And also you can put 25 and 26 lines in one line